tools/mpy-tool.py

```diff
@@ -1765,7 +1765,7 @@ def copy_section(file, offset, offset2):
             f.write(merged_mpy)
 
 
-def main():
+def main(args=None):
```

Comment:
This doesn't look like it's related to MicroPython compatibility.  What's it needed for?  Can it be in a separate commit?